e2e/git-workspace-ondemand-smoke-test.sh

```
@@ -0,0 +1,1068 @@
+      find "$RUN_ROOT" -type f -name 'mount-*.log' -print | while read -r log_file; do
+        dump_mount_log "$log_file"
+      done
+    fi
+  elif [ "$GIT_ONDEMAND_KEEP_ARTIFACTS" != "1" ] && [ -n "$RUN_ROOT" ]; then
+    rm -rf "$RUN_ROOT"
+  fi
+  if [ -n "${CLI_BIN:-}" ] && [ -f "$CLI_BIN" ] && [ "$CLI_SOURCE" = "build" ]; then
+    # temp binary from mktemp
+    rm -f "$CLI_BIN" 2>/dev/null || true
+  fi
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)